package.json

```diff
@@ -17,7 +17,7 @@
     "okapiInterfaces": {
       "users": "15.0 16.0",
       "feesfines": "16.0 17.0 18.0 19.0",
-      "data-export-spring": "1.0"
+      "data-export-spring": "1.0 2.0"
```

Comment:
I think it can work only with 2.0, old implementation can work with 1.0, new implementation with 2.0 - that will be merged later